daffodil-core/src/main/scala/org/apache/daffodil/grammar/primitives/PrimitivesTextNumber.scala

```diff
@@ -186,41 +305,57 @@ trait ConvertTextNumberMixin {
    * but cannot be used as an operational pattern given that
    * potentially lots of things have been removed from it.
    */
-  final protected lazy val patternStripped = {
+  final protected lazy val patternNoQuoted = {
```

Comment:
Did you mean `patternNoQuotes`, which would tell the programmer more clearly that all quotes will be removed, not just one pair of quotes?